--- a/include/linux/susfs_def.h
+++ b/include/linux/susfs_def.h
@@ -25,4 +25,16 @@
 #define CMD_SUSFS_IS_SUS_SU_READY 0x555f0
 #define CMD_SUSFS_SUS_SU 0x60000
+#define CMD_SUSFS_ADD_SUS_PATH_LOOP 0x55553
+#define CMD_SUSFS_HIDE_SUS_MNTS_FOR_NON_SU_PROCS 0x55561
+#define CMD_SUSFS_ENABLE_AVC_LOG_SPOOFING 0x60010
+#define CMD_SUSFS_ADD_SUS_MAP 0x60020
+
+#include <linux/compiler.h>
+static inline int susfs_add_sus_path_loop(void __user **arg) { return 0; }
+static inline int susfs_set_hide_sus_mnts_for_non_su_procs(void __user **arg) { return 0; }
+static inline int susfs_add_sus_map(void __user **arg) { return 0; }
+static inline int susfs_set_avc_log_spoofing(void __user **arg) { return 0; }
 
 #define SUSFS_MAX_LEN_PATHNAME 256
